Backend/dev/migrations/dynamic-offer-driver-app/0275-adding-new-issue-flow-related-data.sql

```diff
@@ -0,0 +1,139 @@
+ALTER TABLE atlas_driver_offer_bpp.issue_report RENAME COLUMN driver_id TO person_id;
```

Comment:
Not backward compatible.

Reply:
Changed driver_id to maybe type, and added person_id, for customer driver_id will have null values.